CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased](https://github.com/digitalocean/droplet-agent/tree/HEAD)
 
+## [1.2.1](https://github.com/digitalocean/droplet-agent/tree/1.2.1) (2022-03-28)
+### Updated
+- Added support for validating SSH Keys on a key-by-key basis.
+- Updating SSH Keys will now partially succeed with an input of a mix of valid and invalid SSHKeys.
```

Comment:
I'm not sure if these statements are accurate. 

We were already validating SSH Keys on a key-by-key basis. We were requiring *ALL* keys to be valid, but now we will ignore invalid key. 

I would suggest a paragraph like:
- Update ssh keys will ignore invalid keys. We noticed some previous keys configured in a droplet may became deprecated by OpenSSH and causing the validation failed. We changed the requirement to allow skipping those invalid keys and only process the valid keys.

Reply:
thank you for the feedback! I updated the text in that section to reflect your suggestions.